models/issue.go

```diff
@@ -779,41 +779,48 @@ type IssueStatsOptions struct {
 // GetIssueStats returns issue statistic information by given conditions.
 func GetIssueStats(opts *IssueStatsOptions) *IssueStats {
 	stats := &IssueStats{}
+	var queryParams []interface{}
+	var condParams []interface{}
 
 	queryStr := "SELECT COUNT(*) FROM `issue` "
 	if opts.LabelID > 0 {
-		queryStr += "INNER JOIN `issue_label` ON `issue`.id=`issue_label`.issue_id AND `issue_label`.label_id=" + com.ToStr(opts.LabelID)
+		queryParams = append(queryParams, opts.LabelID)
+		queryStr += "INNER JOIN `issue_label` ON `issue`.id=`issue_label`.issue_id AND `issue_label`.label_id=?"
 	}
 
-	baseCond := " WHERE issue.repo_id=" + com.ToStr(opts.RepoID) + " AND issue.is_closed=?"
+	condParams = append(condParams, opts.RepoID)
+	baseCond := " WHERE issue.is_closed=? AND issue.repo_id=?"
```

Comment:
No, I have not missed it. It's a more or less dynamic parameter that gets set below in the switch. I just moved it to the beginning to be able to use the slice for the other parameters.